src/main/java/storage/InputStorage.java

```diff
@@ -0,0 +1,15 @@
+package storage;
+
+import java.util.List;
+
+public class InputStorage {
+    private static List<String> inputData;
+
+    public static List<String> getInputData() {
+        return inputData;
+    }
+
+    public static void setInputData(List<String> inputData) {
+        InputStorage.inputData = inputData;
+    }
```

Comment:
we don't need to store some intermediate data, we can just send it straight to the strategy to handle it and then use already valid data in the report, so lets remove this

Reply:
That was for checks ands filters, but you say it has no sense, so ok)